apple/ViewManagers/RNSVGSvgViewManager.mm

```diff
@@ -29,14 +29,8 @@ - (RNSVGView *)view
 RCT_EXPORT_VIEW_PROPERTY(vbHeight, CGFloat)
 RCT_EXPORT_VIEW_PROPERTY(align, NSString)
 RCT_EXPORT_VIEW_PROPERTY(meetOrSlice, RNSVGVBMOS)
-RCT_CUSTOM_VIEW_PROPERTY(tintColor, id, RNSVGSvgView)
-{
-  view.tintColor = [RCTConvert UIColor:json];
-}
-RCT_CUSTOM_VIEW_PROPERTY(color, id, RNSVGSvgView)
-{
-  view.tintColor = [RCTConvert UIColor:json];
-}
+RCT_EXPORT_VIEW_PROPERTY(tintColor, UIColor)
```

Comment:
Those were treated in a custom way for some reason, but this works on both `ios` and `macos`.